cmpctircd/SocketListener.cs

```diff
@@ -43,7 +45,7 @@ public class SocketListener {
         }
         public virtual void Stop() {
             if (_started) {
-                _ircd.Log.Debug($"Shutting down listener [IP: {Info.Host}, Port: {Info.Port}, TLS: {Info.IsTls}]");
+                log.Debug($"Shutting down listener [IP: {Info.Host}, Port: {Info.Port}, TLS: {Info.IsTls}]");
```

Comment:
I can already see that this is so much better than passing around everything inside IRCd